ivy/utils/backend/sub_backend_handler.py

```diff
@@ -57,21 +57,21 @@ def set_sub_backend(sub_backend_str: str):
 
     if ivy.current_backend_str() not in _backend_to_sub_backends_dict.keys():
         logging.warn(
-            f"backend {ivy.current_backend_str()}\
-                  does not have any supported sub_backends"
+            f"backend {ivy.current_backend_str()}                  does not have any"
```

Comment:
I think ideally should be:
```suggestion
            f"backend {ivy.current_backend_str()} does not have any" \
            " supported sub_backends"
```